src/service/poc_mobile.proto

```diff
@@ -692,6 +700,25 @@ message invalid_data_transfer_ingest_report_v1 {
   uint64 timestamp = 3;
 }
 
+message pending_data_transfer_session_v1 {
+  // Public Key of the Hotspot
+  bytes pub_key = 1;
```

Comment:
That's a good callout. 
They are named this way because most of the fields were lifted directly from `data_transfer_event`. 

Because of how tightly these two messages are related, I think it would be equally as confusing to have different field names between them.